src/Dev/Deprecation.php

```diff
@@ -294,6 +367,7 @@ public static function notice($atVersion, $string = '', $scope = Deprecation::SC
                 $data = [
                     'key' => sha1($string),
                     'message' => $string,
+                    'calledFromSupportedCode' => false,
```

Comment:
```suggestion
                    'calledFromSupportedCode' => Deprecation::calledFromSupportedCode($backtrace),
```
If it's called from supported code, it's called from supported code.

Reply:
Have updated to leave a comment about this

There's been corresponding work done in https://github.com/silverstripe/silverstripe-config/pull/124 to ensure that we're not emit deprecatation notices from yaml defined in supported modules